### PR TITLE
Bw 670

### PR DESCRIPTION
**Pull request by milansar**

### CODE REVIEW

File: App_Resources/iOS/Info.plist [hunk 1]
```diff
@@ -16,11 +15,11 @@
 	<key>CFBundlePackageType</key>
 	<string>APPL</string>
 	<key>CFBundleShortVersionString</key>
-	<string>1.0</string>
+	<string>1.1</string>
```
Comment (milansar) [inline App_Resources/iOS/Info.plist]: i think this is modified by CLI and should be place holder

File: App_Resources/iOS/Info.plist [hunk 2]
```diff
@@ -16,11 +15,11 @@
...
 	<key>CFBundleSignature</key>
 	<string>????</string>
 	<key>CFBundleVersion</key>
-	<string>28.0</string>
+	<string>42</string>
 	<key>LSRequiresIPhoneOS</key>
 	<true/>
 	<key>UILaunchStoryboardName</key>
```
Comment (milansar) [inline App_Resources/iOS/Info.plist]: i think this is modified by CLI and should be place holder

File: configurations/trivia/ios/Info.plist.dev [hunk 3]
```diff
@@ -15,11 +15,11 @@
 	<key>CFBundlePackageType</key>
 	<string>APPL</string>
 	<key>CFBundleShortVersionString</key>
-	<string>${versionName}</string>
+	<string>1.1</string>
```
Comment (milansar) [inline configurations/trivia/ios/Info.plist.dev]: i think this is modified by CLI and should be place holder

File: configurations/trivia/ios/Info.plist.dev [hunk 4]
```diff
@@ -15,11 +15,11 @@
...
 	<key>CFBundleSignature</key>
 	<string>????</string>
 	<key>CFBundleVersion</key>
-	<string>${versionCode}</string>
+	<string>42</string>
```
Comment (milansar) [inline configurations/trivia/ios/Info.plist.dev]: i think this is modified by CLI and should be place holder

File: projects/shared-library/src/lib/core/services/mobile/navigation.service.ts [hunk 5]
```diff
@@ -17,22 +17,30 @@ export class NavigationService {
   }
 
   back() {
-    if (this.router.url === '/user/my/invite-friends' ||
-      this.router.url === '/user/my/questions' ||
-      this.router.url === '/recent-games' ||
+  //  console.log('this.router.url------>', this.router.url);
+    if (this.router.url === '/dashboard/leaderboard' ||
```
Comment (milansar) [inline projects/shared-library/src/lib/core/services/mobile/navigation.service.ts]: lets keep leaderboard route to /leaderboard only, it is not part of dashboard

**Comment by milansar** [comment]
@anihalaney , PR is ready for your review

File: package.json [hunk 6]
```diff
@@ -76,7 +76,7 @@
     "nativescript-imagecropper": "1.0.5",
     "nativescript-imagepicker": "^6.2.0",
     "nativescript-iqkeyboardmanager": "^1.5.1",
-    "nativescript-plugin-firebase": "^10.0.0",
+    "nativescript-plugin-firebase": "^9.1.1",
```
Comment (anihalaney) [inline package.json]: reverting back to prior version?

Comment (yugank1991) [reply package.json]: There is issue of conflicts .

**Pull request merged**